.gitignore

```diff
@@ -1,2 +1 @@
-.env
-.idea
\ No newline at end of file
+/docker/keycloak/hsperfdata_jboss/*
```

Comment:
You can't (easily) check in an empty folder using Git, but still:
```suggestion
/docker/keycloak/hsperfdata_jboss
```

Reply:
As we're referencing `.env` in documentation and most developers will use it, it's okay to keep `.env` in the .gitignore.

Reply:
done